.circleci/config.yml

```diff
@@ -169,18 +174,21 @@ workflows:
           xcode-version: '11.7.0'
           ios-sim: 'platform=iOS Simulator,name=iPhone 8,OS=12.4'
           build-universal: false
+      - build_linux:
+          name: Linux - Swift 5.5
+          docker-image: swift:5.5
       - build_linux:
           name: Linux - Swift 5.4
-          docker-image: norionomura/swift:5.4
```

Comment:
There isn't a 5.5 version for this image, so it felt like a good time to switch to the official images.

Reply:
Was there some reason we were using this one?

Reply:
No mention of why it was chosen originally. Replacing it just worked out of the box.